### DISCUSSION TITLE
Atom - PHP - Strange problem with text editing

### DISCUSSION THREAD

**Discussion by Woof71** [atom/atom#22506] [Q&A]
Hi, I have a strange problem with my file.

Look at the pic: it appears that the cursor is shown in a different position than where it is working.

If I click on some lines the cursor disappears, if I press enter in some positions, it returns incorrectly.

I don’t know if it is caused by some setting I did or it is a bug.

Can you help me to solve?

Thank you!

![atom-delirium](https://user-images.githubusercontent.com/85014353/120065806-92b46500-c073-11eb-88a3-a0056655930d.gif)

**Comment by Fleshreg** [comment]
![image](https://user-images.githubusercontent.com/77539267/120077049-1f314880-c0b1-11eb-9387-646a11cfec9d.jpeg)

**Reply by Woof71** [reply]
![Rospo-NO](https://user-images.githubusercontent.com/85014353/120078119-302c8a80-c0ae-11eb-896f-ade797e9626e.jpeg)
